src/webp_converter.js

```diff
@@ -19,21 +21,31 @@ const path = require('path');
         });
 
         const files = await readdir(dirPath);
-
-        // Find directories in current directory and convert their images to webp
+        // Search through the files in current dir for more dirs
+        // Call this function on any found dirs
         for (const file of files) {
             const isDirectory = await (await stat(`${path.join(path.resolve(path.dirname('')), dirPath)}/${file}`)).isDirectory();
 
             isDirectory && await convertImagesToWebp(`${dirPath}/${file}`);
         }
     }
 
+    /**
+     * Find and iterate over all the folders in the Content directory
+     * and convert all the images to webp format
+     */
     (async function () {
         try {
             console.log('Webp conversion START');
-            // Start converting from the Docs folder
-            await convertImagesToWebp('docs');
-            console.log('Webp conversion DONE');
+
+            // Get all the content dirs
+            const contentDirs = await getDirectoriesInPath(CONTENT_DIR_NAME);
+
+            // Call the conversion function for each content dir found
+            // It will recursively scan all the files in the given directory and convert them
+            contentDirs.map(async (dir) => {
+                await convertImagesToWebp(dir);
```

Comment:
Oh no, @m-murasovs, promises won't work this way! With map/forEach you would have to use the async package. I'd go with `for (... of`